LocalMultiplayerAgent/MultiplayerSettings_Linux.json

```diff
@@ -0,0 +1,36 @@
+{
+  "RunContainer": true,
+  "OutputFolder": "C:\\output\\UnityServerLinux",
+  "NumHeartBeatsForActivateResponse": 10,
+  "NumHeartBeatsForTerminateResponse": 60,
+  "TitleId": "59F84",
```

Comment:
this file should be excluded before merge

Reply:
deleted